app/app.go

```diff
@@ -174,5 +174,9 @@ func addData(management *config.ManagementContext, cfg Config) error {
 		return err
 	}
 
+	if err := addCattleGlobalNameSpace(management); err != nil {
```

Comment:
should be `Namespace`

Reply:
This is from commit from Prachi's PR, so I'll change this while rebasing with master :)